RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerItemViewModel.cs

```diff
@@ -31,7 +31,8 @@ public class CompareByType : Comparer<CodeExplorerItemViewModel>
     {
         private static readonly Dictionary<DeclarationType, int> SortOrder = new Dictionary<DeclarationType, int>
         {
-            // Some DeclarationTypes we want to treat the same, like Subs and Functions.
+            // Some DeclarationTypes we want to treat the same, like Subs and Functions,
```

Comment:
I'm not sure `Sub` and `Function` should be treated the same, but depending on sorting options... is there a way to have the `Public` members should show up before the `Private` ones?